spec/02-integration/05-proxy/09-websockets_spec.lua

```diff
@@ -2,13 +2,15 @@ local client = require "resty.websocket.client"
 local helpers = require "spec.helpers"
 local cjson = require "cjson"
 
-for _, strategy in helpers.each_strategy() do
-  describe("Websockets [#" .. strategy .. "]", function()
+for _, enable_buffering in ipairs({ false, true }) do
+for _, strategy in ipairs({"postgres"}) do
```

Comment:
Why only postgres?

Reply:
@gszr oh, that was my mistake, my dev cassandra was broken so I disabled, and forgot that in tests.

Reply:
@gszr, force pushed a fix to that.